src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,30 +1,50 @@
-import React, { PropTypes } from 'react';
+/* global FB, location */
```

Comment:
It may be a little clearer to use `window.location` (and `window` is already globally ignored in our `.eslintrc`).  

And why is `FB` a global/where is that coming from?  I'd prefer if we used imports where possible, or if something is attaching it to `window`, we use `window.FB` and make sure it's well commented